CHANGES.md

```diff
@@ -1,6 +1,20 @@
 # Change Log
 
-### 1.87 - 2021-11-1
+### 1.87.1 - 2021-11-09
+
+##### Additions :tada:
+
+- Added experimental class `ModelExperimental`, a new architecture for `Model` to support new glTF extensions.
+- Added experimental implementation of [3D Tiles Next extensions](https://github.com/CesiumGS/3d-tiles/tree/main/next) for 3D Tiles and glTF:
+  - [3DTILES_metadata](https://github.com/CesiumGS/3d-tiles/tree/main/extensions/3DTILES_metadata) for adding structured metadata to tilesets, tiles, or groups of tile content
+  - [3DTILES_multiple_contents](https://github.com/CesiumGS/3d-tiles/tree/main/extensions/3DTILES_multiple_contents) for storing multiple contents within a single tile
+  - (glTF extension) [EXT_mesh_features](https://github.com/KhronosGroup/glTF/pull/2082) for adding semantic metadata to glTF models and tile contents. This is the glTF equivalent of `3DTILES_metadata`. Requires use of `ModelExperimental`.
```

Comment:
```suggestion
  - [EXT_mesh_features](https://github.com/KhronosGroup/glTF/pull/2082) (glTF extension)  for adding semantic metadata to glTF models and tile contents. This is the glTF equivalent of `3DTILES_metadata`. Requires use of `ModelExperimental`.
```

| Before | After |
| - | - |
| ![image](https://user-images.githubusercontent.com/5172619/140779756-4d08116a-74bb-4e74-8f0e-fc30fd464277.png) | ![image](https://user-images.githubusercontent.com/5172619/140779680-26ce79b3-278f-42a6-be36-386246da22a0.png) |

I just think this looks slightly better.